gatsby-theme-landing-page/src/components/base.module.css

```diff
@@ -29,3 +34,20 @@
   margin-left: auto;
   margin-right: auto;
 }
+
+.heading {
+  font-size: var(--font-size-6);
+  margin-bottom: var(--space-2);
```

Comment:
Might be good to go ahead and define default margin top and bottom for both of these

Reply:
Also just thinking out loud: wonder if it makes sense to pull these into components rather than using `composes` 🤔